sensor_msgs/PointCloud2   integrated_cloud
---
string[]                  index_to_label_name
int32[]                   label
float32[]                 label_probabilities
float32[]                 label_frequencies
geometry_msgs/Point32[]   points